### PR TITLE
feat: verus flat fly

### PR DESCRIPTION
**Pull request by liquidsquid1**
adds a simple verus flat fly

the reason this works is verus becomes super dumb when you clip, so blinking with that allows you to infinitely air walk. on top of that, timer works with high pauses inbetween movement packets which means that any timer would work with the blink (as long as you are on 1.9.x or higher)

can rarely flag when moving whilst stopping, but i've set the velocity to 0 to combat that.

tested on anticheat-test AND loyisa

you can only land on the same Y level

### CODE REVIEW

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/fly/modes/verus/FlyVerusFlat.kt [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+    override val parent: ChoiceConfigurable<*>
+        get() = ModuleFly.modes
+
+    var requiresLag = false
+        private set
+        get() = this.handleEvents()
+
+    val packetHandler = handler<PacketEvent> { event ->
+        if (event.packet is PlayerMoveC2SPacket) {
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/fly/modes/verus/FlyVerusFlat.kt]: Put event.packet into extra variable

**Comment by liquidsquid1** [comment]
seems like detekt hates me, will fix tomorrow so that it compiles

**Comment by EclipsesDev** [comment]
very complex condition real

**Comment by liquidsquid1** [comment]
> very complex condition real

has 8 conditions
i add 1 condition
now it says the max is 5?!!?

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/fly/modes/verus/FlyVerusFlat.kt [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+import net.minecraft.util.shape.VoxelShapes
+
+/**
+ * @anticheat Verus
+ * @anticheatVersion b3896
+ * @testedOn anticheat-test
+ * @note it can rarely flag once | needs 1.9x or above
+ */
+
+internal object FlyVerusFlat : Choice("VerusFlat") {
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/fly/modes/verus/FlyVerusFlat.kt]: Put the version name of Verus into the name, then I will merge it.

**Review by 1zun4 — CHANGES_REQUESTED** [review]
please add the version name to the fly name

**Review by 1zun4 — APPROVED** [review]

**Pull request merged**